src/index.html

```diff
@@ -11,7 +11,180 @@
+            name="birth-date"
+          >
+        </label>
+
+        <label class="form-field_last">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field_last">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="8"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              name="Yes"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="No"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="bedtime"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+
+          <select
+            name="cars"
+            required
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field_last">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rating"
+            min="0"
+            max="100"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            autocomplete="off"
+         ></textarea>
+        </label>
+
+        <label class="form-field_last">
+          Would you recommend us?
+
+          <select name="recommend">
+            <option
+              value="yes"
+              selected
+            >
+            </option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button
+        type="submit"
+        class="button">
+        Submit
+      </button>
```

Comment:
```suggestion
      <button
        type="submit"
        class="button"
      >
        Submit
      </button>
```
check [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) pls

Reply:
Change applied, thanks!

Reply:
Something went wrong with re-request.
I approved, but if this problem appears in next your PRs, tag current online mentor in chat.

And just one more comment.
Please,don't forget attach your demo links in PR description in next tasks.